docs/service-brokers/docs/020-architecture-reb.md

```diff
@@ -3,25 +3,17 @@ title: The Remote Environment Broker architecture
 type: Architecture
 ---
 
-The Remote Environment Broker (REB) workflow starts with the registration process, during which a remote environment (RE) is registered on the Kyma cluster.
+The Remote Environment Broker (REB) workflow consists of the following steps:
 
-### Remote environment registration process
+1. The Remote Environment Broker watches for RemoteEnvironments (REs) in the cluster and EnvironmentMappings (EMs) in all Namespaces.
+2. The user creates an EnvironmentMapping custom resource in a given Environment. The EnvironmentMapping activates the services offered by a RemoteEnvironment of the same name.
```

Comment:
"of the same name" sounds like a magic, please clearly say that EM must has same name as RE that will be activated in a given Environment